src/WebExploitation/FileUploads/Content-type restriction.md

```diff
@@ -0,0 +1,89 @@
+# File Uploads: Content-type restriction
+
+## Introduction
+In the wild, it's unlikely that you'll find a website that has no protection against file upload attacks like we saw in the previous lab. But just because defenses are in place, that doesn't mean that they're robust. You can sometimes still exploit flaws in these mechanisms to obtain a web shell for remote code execution. 
```

Comment:
which previous lab is this referring to? As far as I understand, there's been no lab according to gitbook's order of articles